packages/paste-website/src/pages/components/menu/index.mdx

```diff
@@ -114,6 +115,26 @@ Each menu item can only perform a single action.
   {defaultExample}
 </LivePreview>
 
+### Menu with groups
+
+MenuGroups can be given a prefix icon. Please make sure to use icons in a [decorative manner](http://localhost:8000/icon-system#specify-if-an-icon-is-decorative).
```

Comment:
Should we need to explain anything more about `MenuGroups`?

Reply:
What's missing for you?

Reply:
I was wondering if we needed to provide any direction on when to use `MenuGroups`.

Reply:
We could include a one liner around that, something like (don't copy and paste my typos 😉 )

> Use Menu groups to create hierarchy or logical groupings within longer menus. Menu groups have a visible group label that should describe purpose or categorization of the grouping clearly.